index.js

```diff
@@ -28,22 +28,36 @@ const saveAs = (uri, filename) => {
 /**
  * @param  {React.RefObject} node
  * @param  {string} fileName
- * @param  {string} backgroundColor
  * @param  {string} type
- * @param  {object} options=null
+ * @param  {object} html2CanvasOptions=null
  */
-const exportComponent = (node, fileName, backgroundColor, type, options) => {
+
+const getPDF = (type, {x, y, orientation}) => {
+    if(orientation === 'l') {
+        return new JsPDF('l', 'mm', [canvas.width, canvas.height])
+    } else if(orientation === 'p') {
+        return new JsPDF('p', 'mm', [canvas.height, canvas.width]);
+    } else {
+        return canvas.width > canvas.height
+        ? new JsPDF('l', 'mm', [canvas.width, canvas.height])
+        : new JsPDF('p', 'mm', [canvas.height, canvas.width]);
+    }
+    
+}
+
+const exportComponent = ({node, fileName, type, html2CanvasOptions, pdfOptions}) => {
+    if(!node.current) {
+        throw new Error("'node' must be a RefObject")
+    }
     const element = ReactDOM.findDOMNode(node.current);
     return html2canvas(element, {
         backgroundColor: backgroundColor,
         scrollY: -window.scrollY,
         useCORS: true,
-        ...options
+        ...html2CanvasOptions
     }).then(canvas => {
         if (type === fileType.PDF) {
-            const pdf = canvas.width > canvas.height
-                ? new JsPDF('l', 'mm', [canvas.width, canvas.height])
-                : new JsPDF('p', 'mm', [canvas.height, canvas.width]);
+            const pdf = getPDF(type, pdfOptions)
             pdf.addImage(canvas.toDataURL(fileType.PNG, 1.0), 'PNG', 0, 0);
```

Comment:
x, y from pdf options will be used here.

Reply:
i addressed this in the latest commit